beater/config/config.go

```diff
@@ -42,6 +44,45 @@ var (
 	regexObserverVersion = regexp.MustCompile("%.*{.*observer.version.?}")
 )
 
+type kibanaConfig struct {
+	Enabled bool                `config:"enabled"`
+	Config  kibana.ClientConfig `config:",inline"`
```

Comment:
Can `kibana.ClientConfig` be embedded? It would be nice if we could turn (for example) `cfg.Kibana.Config.Host` into `cfg.Kibana.Host`.

Reply:
Great idea.  I was trying to make `type kibanaConfig kibana.ClientConfig` originally and just embedding didn't even occur to me.